### PR TITLE
bpo-46402: Promote SQLite URI tricks in `sqlite3` docs

### PR DESCRIPTION
**Pull request by erlend-aasland**
https://bugs.python.org/issue46402

### CODE REVIEW

File: Doc/library/sqlite3.rst [hunk 1]
```diff
@@ -273,14 +273,28 @@ Module functions and constants
...
-   to specify options. For example, to open a database in read-only mode
-   you can use::
-
-       db = sqlite3.connect('file:path/to/database?mode=ro', uri=True)
-
-   More information about this feature, including a list of recognized options, can
-   be found in the `SQLite URI documentation <https://www.sqlite.org/uri.html>`_.
+   If *uri* is :const:`True`, *database* is interpreted as a
+   :abbr:`URI (Uniform Resource Identifier)` consisting of a scheme, an
+   authority, a path, and a query string. The scheme part *must* be ``"file:"``.
```
Comment (nedbat) [inline Doc/library/sqlite3.rst]: The enumeration of URI components is either redundant with the understood form of file URIs, or potentially confusing ("Is this different than my understanding of URIs? What's an authority?").  Better would be to explain how to express a file path as a URI:
```suggestion
   :abbr:`URI (Uniform Resource Identifier)` with a file path and an optional query string.
   The scheme part *must* be ``"file:"``.  The path can be a relative or absolute file path.
```

**Comment by nedbat** [comment]
This is great, thanks!  I don't know if my vote counts for anything, but I approve. :)

File: Doc/library/sqlite3.rst (hunk 1 above)
Comment (erlend-aasland) [reply Doc/library/sqlite3.rst]: I agree that the authority part is confusing. We already link to the SQLite docs for those who want to explore this feature further. Lets keep it simple :)

**Comment by erlend-aasland** [comment]
Thanks for reviewing, @nedbat :)

**Comment by ericvsmith** [comment]
I think this looks great.

**Review by ericvsmith — APPROVED** [review]

**Comment by erlend-aasland** [comment]
Thanks, Eric!

**Comment by erlend-aasland** [comment]
BTW, would you mind merging it? I don't have the commit bit.

**Comment by ericvsmith** [comment]
Will do.

**Comment by erlend-aasland** [comment]
cc. @JulienPalard: I only need to update suspicious for the backports, right? (`file:` triggers the suspicious checks in 3.10 and 3.9).

**Comment by erlend-aasland** [comment]
FYI: I'm closing the auto-backports and doing them myself in order to get the suspicious fixes in those.

**Pull request merged**